waiter/src/waiter/scheduler.clj

```diff
@@ -76,7 +76,8 @@
    ^String log-directory
    ^String message])
 
-(defn make-ServiceInstance [value-map]
+(defn make-ServiceInstance [{id :id started-at :started-at :as value-map}]
+  (log/log "InstanceTracker" :debug nil (str "Created an instance with id: " id " that started at " started-at))
```

Comment:
We usually prefer creating a map with the fields we're interested in, and then logging the map. Same for extra data included in exception error messages.
```clojure
(log/log "InstanceTracker" :debug (str "Created an instance " {:id id :started-at started-at}))
```